src/esm.ts

```diff
@@ -72,6 +78,35 @@ export function registerAndCreateEsmHooks(opts?: RegisterOptions) {
     );
   }
 
+  async function load(
+    url: string,
+    context: {},
+    defaultLoad: typeof load
+  ): Promise<{ format: Format; source: string | Buffer }> {
+    const { format } = await getFormat(url, context, defaultGetFormat);
+
+    const { source: rawSource } = await defaultLoad(
+      url,
+      { format },
+      defaultLoad
+    );
+
+    const defaultTransformSource: typeof transformSource = async (
```

Comment:
It took me a second to understand what's going on here; might benefit from a couple comments.

`defaultTransformSource` is an identity function?  Is that idea that any transformations from *outside* of ts-node, if they happen at all, are going to come from `defaultLoad`?

Reply:
Yes, it is a identity function because the old `defaultTransformSource` inside node was an identity function. 
https://raw.githubusercontent.com/nodejs/node/v15.3.0/lib/internal/modules/esm/transform_source.js
The idea was just to be able to re-use the existing `transformSource` and it required the old `defaultTransformSource` to defer to. Perhaps we should not use the existing functions for the old hooks in `load`?

Reply:
I added some comments in an attempt to clarify.